src/index.js

```diff
@@ -11,7 +9,7 @@ export default class captureVisibleTabFull {
             return base.then(() => {
                 return this._sendMessage(tab, {'type': 'doScroll', index});
             }).then(({top, left}) => {
-                return this._sleep(150).then(() => ({top, left}));
+                return this._sleep(300).then(() => ({top, left}));
```

Comment:
The 300 micro second timeout still results in an "This request exceeds the MAX_CAPTURE_VISIBLE_TAB_CALLS_PER_SECOND quota" error. having it set as a parameter might be ideal

Reply:
From the official Chrome extension [documentation](https://developer.chrome.com/docs/extensions/reference/api/tabs#property-MAX_CAPTURE_VISIBLE_TAB_CALLS_PER_SECOND), `captureVisibleTab ` can only be called 2 times a second

![image](https://github.com/kyo-ago/chrome-tab-capture-visible-tab-full/assets/19529979/d1d81b07-c4b7-4d6b-950b-0195f894510c)

So the minimal figure should be 500